physics/rigid_motion.hpp

```diff
@@ -76,10 +78,35 @@ RigidMotion<FromFrame, ToFrame> operator*(
     RigidMotion<ThroughFrame, ToFrame> const& left,
     RigidMotion<FromFrame, ThroughFrame> const& right);
 
+// A |RigidTransformation|, its first derivative (a |RigidMotion|), and its
+// second derivative (angular and linear accelerations).
+template<typename FromFrame, typename ToFrame>
+class SecondOrderRigidMotion {
+ public:
+  SecondOrderRigidMotion(
+      RigidMotion<FromFrame, ToFrame> const& first_order_motion,
```

Comment:
`s/first_order_motion/rigid_motion/`

A `RigidMotion` is by definition a first-order thingy.